connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/ExecuteHolder.scala

```diff
@@ -41,6 +42,8 @@ private[connect] class ExecuteHolder(
   val responseObserver: ExecuteResponseObserver[proto.ExecutePlanResponse] =
     new ExecuteResponseObserver[proto.ExecutePlanResponse]()
 
+  val executeEventsManager: ExecuteEventsManager = ExecuteEventsManager(this, new SystemClock())
```

Comment:
ocd: since we're already inside ExecuteHolder, maybe just eventsManager (same as responseObserver above and runner below)